app/models/mail_template.rb

```diff
@@ -0,0 +1,31 @@
+class MailTemplate < ActiveRecord::Base
+  # t.string   :name
+  # t.string   :template_type
+  # t.string   :from
+  # t.string   :to
+  # t.string   :subject
+  # t.string   :attachment
+  # t.text     :text
+  # t.integer  :organization_id
+  # t.timestamps
+
+  TEMPLATE_TYPES = ['invoice', 'reminder']
+
+  attr_accessible :name, :template_type, :from, :to, :subject, :attachment, :text
+
+  belongs_to :organization
+
+  validates :name, presence: true, uniqueness: { scope: :organization_id}
+  validates :template_type, inclusion: { in: TEMPLATE_TYPES }
+  validates :from, presence: true
+  validates :to, presence: true
```

Comment:
When a template is registered, it gives the user a better understanding when the mails from / to is explicitly shown in the form. 
Take the mails from / to from the database may be beneficial on certain occasions.